app/admin/route.js

```diff
@@ -1,12 +1,18 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  session: Ember.inject.service(),
+
   model() {
     return this.store.ajax({
       url: '/api/v1.0/users/me'
     });
   },
-
+  
+  afterModel() {
```

Comment:
afterModel accepts model as parameter. No need to call `this.modelFor(this.routeName))`.